content/services/cloudcoach.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Cloud Coach"
+type: "services"
+subtitle: ""
+description: "We help you be successful in the Cloud by being involved advisors and sparring partners. We will also help you should things go wrong."
+icon: developer.svg
+---
+
+{{<section>}}
+{{% col-left-10 title="Cloud Coach" %}}
+There is great value in cloud technology for startups, but having a dedicated skilled Platform Team often isn’t financially viable until they become scale-ups. With our Cloud Coach service, we help your engineers leverage cloud technology effectively by being involved advisors and sparring partners. We help you make the right decisions, gather the right skills, and build the right things the right way; all while also offering active support should things go wrong.
+
+
+Our Cloud Coach service helps startups be successful in using cloud technology in a financially sustainable way. Our service consists of 2 parts;
+
+- Hands-off Coaching
+- Slack Support
+
+{{% /col-left-10 %}}
+
+{{<col-right-2>}}
+{{<img class="img-fluid" src="/img/icons/developer.svg">}}
+{{</col-right-2>}}
+{{</section>}}
+
+{{<section>}}
+{{<col-left-2>}}
+{{<img class="img-fluid" src="/img/icons/consultant.svg">}}
+{{</col-left-2>}}
+
+{{% col-right-10 title="Hands-off Coaching" %}}
+
+Our Cloud Coaching approach is a _hands-off_ approach. This means we are here to offer guidance, advice, training, pair-programming, or review, all while you remain in control of your own setup. Periodically planned coaching sessions mean we stay involved in your current developments, and we can tackle current and upcoming challenges together.
+
+When needed we can schedule extra sessions for deep-dives, training, proof-of-value setups or pair-programming.
+
+{{% /col-right-10 %}}
+
+{{</section>}}
+
+
+{{<section>}}
+{{% col-left-10 title="Engineering-friendly Support" %}}
+
+Our Slack Support offers direct access to our team of experts for quick questions, or for active help should things go wrong. If your latest deployment took down your website, just ping us on Slack and we will debug *with* you. Every Cloud Coach client gets their own private Slack Connect channel that works right in your existing Slack workspace. It's convenient, and we are engineers too.
```

Comment:
"It's convenient, and we are engineers too."

I feel like that last sentence should probably go unsaid, it should be self-evident that we're engineers if we're helping them out with their issues. The paragraph until that sentence also stands strong enough on its own that this sentence probably doesn't need to be here.